index.vwf.html

```diff
@@ -30,11 +30,24 @@
 <script type="text/javascript" src="source/view/videoController.js"></script>
 <script type="text/javascript" src="source/view/fileManager.js"></script>
 
+
 <!-- CSS -->
 <link rel="stylesheet" href="source/view/index.css"></link>
 <link rel="stylesheet" href="source/view/jquery-resizable.css"></link>
+<link type="text/css" href="minimal_jplayer_style.css" rel="stylesheet" />
 
 <!-- HTML -->
+<div id="jp_container_1" class="jp-video jp-video-360p">
+
+    <!--container in which our video will be played-->
+    <div id="jquery_jplayer_1" class="jp-jplayer"></div>
+ 
+    <!--main containers for our controls-->
+    <div class="jp-gui">
+        <div class="jp-interface">
+        </div>
+    </div> 
+</div>        
 <div id="version"></div>
```

Comment:
@scottnc27603 Right now there's a dependency on the correct divs in index.vwf.html. This is also true of TDG, however, which uses the current, upstream jPlayer driver. I do not see this as a regression.